include/openssl/ocsp.h

```diff
@@ -242,6 +258,20 @@ OPENSSL_EXPORT OCSP_CERTID *OCSP_cert_to_id(const EVP_MD *dgst,
 OPENSSL_EXPORT int OCSP_parse_url(const char *url, char **phost, char **pport,
                                   char **ppath, int *pssl);
 
+// OCSP_id_cmp compares the contents of |OCSP_CERTID|, returns 0 on equal.
```

Comment:
It follows the standard `< 0` if `a<b` and `>0` if `a>b`. The underlying function compares multiple fields (the hash algorithm type, `issuerNameHash`, `issuerKeyHash`, and the `serialNumber`) to check if they are equal.
I had considered documenting the circumstances of "less than or greater than", but I figured this didn't really matter since people using `OCSP_id_cmp` are checking for an exact match, rather than trying to see which specific field is larger.